fvm/fvm_test.go

```diff
@@ -1948,6 +1948,25 @@ func TestHashing(t *testing.T) {
 				require.Equal(t, "f41e8de9af0c1f46fc56d5a776f1bd500530879a85f3b904821810295927e13a54f3e936dddb84669021052eb12966c3", result)
 			},
 		},
+		{
+			Algo:    runtime.HashAlgorithmKECCAK_256,
+			WithTag: false,
+			Check: func(t *testing.T, result string, scriptErr errors.Error, executionErr error) {
+				require.NoError(t, scriptErr)
+				require.NoError(t, executionErr)
+				require.Equal(t, "1d5ced4738dd4e0bb4628dad7a7b59b8e339a75ece97a4ad004773a49ed7b5bc", result)
+			},
+		},
+		{
+			Algo:    runtime.HashAlgorithmKECCAK_256,
+			WithTag: true,
+			Tag:     "some_tag",
+			Check: func(t *testing.T, result string, scriptErr errors.Error, executionErr error) {
+				require.NoError(t, scriptErr)
+				require.NoError(t, executionErr)
+				require.Equal(t, "8454ec77f76b229a473770c91e3ea6e7e852416d747805215d15d53bdc56ce5f", result)
```

Comment:
I was not 100% sure how to compute Keccak 256 reference hash with some tags, @tarakby could you please check the correctness of this reference.

Reply:
You should be able to use this online tool: https://emn178.github.io/online-tools/keccak_256.html

Reply:
Yeah but how could you pass the tag, is it just prepending the tag to the data?

Reply:
Yes, for Keccak it is just prepending the message with the tag